python/ctsm/site_and_regional/plumber2_surf_wrapper.py

```diff
@@ -23,15 +23,21 @@
 import argparse
 import logging
 import os
-import subprocess
+import sys
 import tqdm
 
-import pandas as pd
+# Get the ctsm tools
```

Comment:
We talked about this a bit, and it's not too annoying, but I think it might be uneccessary.

This should be handled in both the testing framework and the top level skeleton scripts with path_utils type things.

Looking at it: plumber2_surf_wrapper does this same thing. And run_ctsm_py_tests does it as well.

So we only need this if we want to support running this standalone outside of those contexts. Is that something you are doing? And that you find useful? 

I think I side on not doing this in individual python modules and just assume it'll be handled by either the top level driver script, or the run_ctsm_py_tests environment. Maintaining this for all python module files seems to cumbersome to me. But, I'm happy to hear other thoughts on it as well.

Reply:
Good point; deleting.

Reply:
Done with 56a36f1ad.